tests/check/check_h26xsigned_sign.c

```diff
@@ -544,10 +543,7 @@ START_TEST(undefined_nalu_in_sequence)
 END_TEST
 
 /* Test description
- * Verify that metadata is sent with recurrence interval equal to 3.
- * Recurrence equal to 1 means that all metadata is sent in every GOP.
- * Recurrence equal to 3 means that all metadata is sent in every second GOP.
- * Metadata that can be sent more rarily are public key and product info.
+ * Verify that metadata is sent with correct recurrence interval in frames on the average.
  * The operation is as follows:
  * 1. Generate a nalu_list with a sequence of signed GOPs.
  * 2. Check the sequence of NALUs.
```

Comment:
Change text of 3. below. We no longer check the size. We check if the SEI has PUBLIC_KEY_TAG, which is recurrent data.